dataset: 'kitti'

cat_list: ['Car']
# cat_list: ['Pedestrian']
#cat_list: ['Car', 'Pedestrian']


Car:
  metric_3d: iou_3d
  metric_2d: iou_2d
  max_ages: 25
  min_frames: 3
  cost_function:
      iou_2d: 0.5
      sdiou_2d: 1.1
      giou_3d: -0.2
      iou_3d: 0.01
      dist_3d: 1.1

Pedestrian:
  metric_3d: iou_3d
  metric_2d: iou_2d
  max_ages: 25
  min_frames: 3
  cost_function:
      iou_2d: 0.5
      sdiou_2d: 0.2
      giou_3d: -0.2
      iou_3d: 0.01
      dist_3d: 1.1

